.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+extends: airbnb-base
+parser: babel-eslint
+env:
+  browser: true
+  node: true
+
+globals:
+  atom: true
+
+settings:
+  import/core-modules: [ atom ]
+
+rules:
+  no-console: [0]
+  no-use-before-define: [0]
+  no-param-reassign: [0]
+  no-underscore-dangle: [0]
+  no-useless-escape: [0]
+  prefer-rest-params: [0]
+  class-methods-use-this: [0]
```

Comment:
> I'm going to remove this again and use static methods as you mentioned above.

My bad. This won't work.

I'm using this rule mainly because it would arror in files like [kernel.js] since a lot of the methods don't use `this`.